packages/docs/docs/tutorials/data-modeling/family-relationships.md

```diff
@@ -0,0 +1,278 @@
+| :-: | :----------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------- | --------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------- |
+|  1  | [**Person**](/docs/api/fhir/resources/patient) with Contact Information                                                                                      | No                                                    | Rarely                                        | Only Patient                                                                                                   | Individual                                                            |
+|  2  | [**Person**](/docs/api/fhir/resources/patient)+ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson)                                                  | Yes                                                   | Rarely                                        | Only Patient                                                                                                   | Individual                                                            |
+|  3  | [**Patient**](/docs/api/fhir/resources/patient)/ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) /[**Person**](/docs/api/fhir/resources/person)  | Yes                                                   | Frequently                                    | Only Patient                                                                                                   | Individual                                                            |
+|  4  | [**Patient**](/docs/api/fhir/resources/patient)/ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) /[**Person**](/docs/api/fhir/resources/patient) | Yes                                                   | Frequently                                    | All Members                                                                                                    | Individual                                                            |
+|  5  | [**Group**](/docs/api/fhir/resources/group) of Patients                                                                                                      | Yes                                                   | Frequently                                    | All Members                                                                                                    | Family Unit                                                           |
+
+- First, you'll have to ask yourself a few questions
+  1. Do family members participate in clinical activiites?
```

Comment:
You should add a note about the CMS queries here - there are many that relate to pediatric use cases.  From a quick skim of the queries it seems like there are maternal screening, PPD and other scenarios that warrant treating parents as patients for reporting purposes https://www.medplum.com/docs/compliance/onc#materials-and-usage